javatime/src/main/java/drewhamilton/androiddatetimeformatters/javatime/AndroidDateTimeFormatter.java

```diff
@@ -13,22 +14,53 @@
 
 import androidx.annotation.RequiresApi;
 
+import static android.text.format.DateFormat.*;
```

Comment:
With all the different "format" classes flying around here, I'd prefer to keep each reference explicit.